DESCRIPTION

```diff
@@ -1,12 +1,18 @@
-Package: r-epigraphhub
+Package: epigraphhubR
```

Comment:
it would be better to have the package name changing in a different PR.
but I think that the package name should be `epigraphhub` instead of `epigraphhubR`. we don´t need this extra `R` in the name.

Reply:
The `r-epigraphhub.Rproj` can also be renamed to just `epigraphhub.Rproj`